types/block_bench_test.go

```diff
@@ -0,0 +1,22 @@
+package types
+
+import (
+	"testing"
+
+	"github.com/NebulousLabs/Sia/encoding"
+)
+
+// BenchmarkDecodeEmptyBlock benchmarks decoding an empty block.
+//
+// David's Computer, 08-20-2015: 38 MB/s
```

Comment:
I waffled on that one. but afaik the processor means more than any of the other hardware. Windows vs. Linux vs. Mac might mean something.